controllers/util/clusters.go

```diff
@@ -157,3 +158,23 @@ func (c *Clusters) HasMultipleStorageClustersWithSameName(name string) bool {
 
 	return count > 1
 }
+
+func IsMultus(nwSpec *rookCephv1.NetworkSpec) bool {
```

Comment:
Good to add a comment for this one as well as its exposed now

Reply:
done